### DISCUSSION TITLE
Several contact sensors delay

### DISCUSSION THREAD

**Discussion by dimkagithub** [merdok/homebridge-miot#444] [Q&A] [answered]
Hi!
I have four Xiaomi Door & Window Sensors.
Every 5-10 minutes I get this error and sensors works with lag:

`[Sensor] Poll failed 4 times in a row! Stopping polling and trying to reconnect! Reason: Error: No response or response empty!`

Then it connects and works for about next 5-10 minutes.

PollingInterval set to 1
Each sensor is connected to the gateway with its own deviceid.

When 1 sensor was connected, there were no problems.

What could be wrong? Too many sensors with frequent sensor polling?

**Comment by merdok** [comment, marked as answer]
A polling interval of 1 second is quite heavy on the device and i am certain that you do not need to poll your devices that often. I would recommend lowering that to at least 5 seconds, 10 would be even better.

**Reply by dimkagithub** [reply]
I agree. But for this sensor, polling should be as often as possible. )
A door or window can open and close within 5 seconds, and polling occurs once every 10 seconds therefore sensor will not be triggered.

PS All other devices poling set to 15.

**Reply by merdok** [reply]
There is nothing I can do about, that's how it is. 
It might have worked when you had 1 sensor connected because there was no load on the gateway. Now that you have more sensors there is a greater load on the gateway hence this is happening. You need to reduce the polling interval if you want to prevent it.

**Reply by dimkagithub** [reply]
Ok. Its clear.
